implementations/rust/ockam/ockam_examples/example_projects/tunnel/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hmm, unless I'm missing something (possible), I think this file probably doesn't need to exist.